lib/js/dom/tags.js

```diff
@@ -0,0 +1,9 @@
+export default [
+  'h3',
+  'div',
+  'input',
+  'section',
+  'button',
+  'textarea',
+  'pre'
+];
```

Comment:
Whoops, no. That was left over from a session where I was tring out `rdom` and was not intended to be committed. It did remind me to declare a `pre` tag for the console / error output, as per the original repl DOM.